src/App.js

```diff
@@ -5,13 +5,15 @@ import './App.css'
 import SignupPage from './pages/SignupPage'
 import Dashboard from './pages/Dashboard'
 import LoginPage from './pages/LoginPage'
+import ProtectedRoute from './components/services/ProtectedRoute'
 
 const App = () => {
   return (
     <BrowserRouter>
       <Routes>
         <Route path="/" element={<LoginPage/>} />
-        <Route path='dashboard' element={<Dashboard/>}/>
+
+        <Route path='dashboard' element={<ProtectedRoute><Dashboard/></ProtectedRoute>}/>
```

Comment:
am impressed with this protected route thing...but you have to make it more intuitive...
There other react concepts that can fix this..
We can look at that later